Composite/Core/PackageSystem/PackageFragmentInstallers/DataPackageFragmentInstaller.cs

```diff
@@ -281,12 +284,15 @@ private void UpdateVersionId(IVersioned data)
         }
 
 
-        private static DataKeyPropertyCollection CopyFieldValues(DataType dataType, IData data, XElement addElement)
+        private static DataPropertyValueCollection GetKeyPropertyValues(DataType dataType, IData data, XElement addElement)

```

Comment:
Shouldn't it be called "CopyKeyPropertyValues" or "PopulateKeyPropertyValues", as Get implies it should change its parameters

Reply:
Yeah, the method both returns the list of KeyProperties and populates them with values from the addElement-argument.

So maybe `PopulateAndReturnKeyPropertyValues`